src/CosmosDB/CosmosDB/CosmosDBAccount/UpdateAzCosmosDBAccount.cs

```diff
@@ -140,12 +140,42 @@ public override void ExecuteCmdlet()
                 databaseAccountUpdateParameters.IpRules = base.PopulateIpRules(IpRule);
             }
 
+            if (ServerVersion != null)
+            {
+                if (databaseAccountUpdateParameters.ApiProperties == null)
+                {
+                    databaseAccountUpdateParameters.ApiProperties = new ApiProperties();
+                }
+
+                databaseAccountUpdateParameters.ApiProperties.ServerVersion = ServerVersion;
+            }
+
             if (NetworkAclBypassResourceId != null)
             {
                 Collection<string> networkAclBypassResourceId = new Collection<string>(NetworkAclBypassResourceId);
                 databaseAccountUpdateParameters.NetworkAclBypassResourceIds = networkAclBypassResourceId;
             }
 
+            if (BackupIntervalInMinutes.HasValue || BackupRetentionIntervalInHours.HasValue)
+            {
+                if (readDatabase.BackupPolicy is PeriodicModeBackupPolicy)
+                {
+                    PSBackupPolicy backupPolicy = new PSBackupPolicy()
```

Comment:
why are we not directly creating BackupPolicy object ?

Reply:
Done. Creating the backupPolicy directly.